config/sidekiq.yml

```diff
@@ -0,0 +1,14 @@
+verbose: true
+logfile: ./log/sidekiq.log
+development:
+  :concurrency: 2
+staging:
+  :concurrency: 2
+demo:
+  :concurrency: 2
+sandbox:
+  :concurrency: 2
+production:
+  :concurrency: 2
+:queues:
+  - database
```

Comment:
I'm having trouble running the background jobs, could be that I just don't know how to use active job properly, so can you check I'm doing this right: I started sidekiq with this config file and clicked "run importer", I can see in the web panel that the job is enqueued in the "default" queue, but that queue is not being processed by sidekiq as it's pulling from "database" queue.

Reply:
Again, thanks for catching that. I was doing some changes in the workers yesterday and I accidentally deleted that line `sidekiq_options queue: :database`.
Now, it's going to the correct queue.